src/Core/Framework/DataAbstractionLayer/FieldSerializer/StateMachineStateFieldSerializer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+// Copyright (c) Pickware GmbH. All rights reserved.
```

Comment:
Copyright header :)

Reply:
Ahh, right, thx :D Anyway, this PR is still in draft, so not ready for review